### PR TITLE
clean up some of the docs, fixing outdated information and other minor issues

### PR DESCRIPTION
**Pull request by iansimon**

### CODE REVIEW

File: magenta/pipelines/README.md [hunk 1]
```diff
@@ -4,10 +4,12 @@ Magenta has a lot of different models which require different types of inputs. S
...
-* [pipelines_common.py](https://github.com/tensorflow/magenta/blob/master/magenta/pipelines/pipelines_common.py) contains some `Pipeline` implementations that convert to common data types, like `Melody`.
-* [dag_pipeline.py](https://github.com/tensorflow/magenta/blob/master/magenta/pipelines/dag_pipeline.py) defines a `Pipeline` which connects arbitrary pipelines together inside it. These `Pipelines` can be connected into any directed acyclic graph (DAG).
-* [statistics.py](https://github.com/tensorflow/magenta/blob/master/magenta/pipelines/statistics.py) defines the `Statistic` abstract class and implementations. Statistics are useful for reporting about data processing.
+* [pipeline.py](/magenta/pipelines/pipeline.py) defines the `Pipeline` abstract class and utility functions for running a `Pipeline` instance.
+* [pipelines_common.py](/magenta/pipelines/pipelines_common.py) contains some `Pipeline` implementations that convert to common data types, like `Melody`.
+* [dag_pipeline.py](/magenta/pipelines/dag_pipeline.py) defines a `Pipeline` which connects arbitrary pipelines together inside it. These `Pipelines` can be connected into any directed acyclic graph (DAG).
```
Comment (adarob) [inline magenta/pipelines/README.md]: defines a `DAGPipeline`

File: magenta/scripts/README.md [hunk 2]
```diff
@@ -4,19 +4,20 @@ After [installing Magenta](/README.md), you can build your first MIDI dataset. W
...
 ```
-MIDI_DIRECTORY=<folder containing MIDI files. can have child folders.>
+INPUT_DIRECTORY=<folder containing MIDI and/or MusicXML files. can have child folders.>
 
 # TFRecord file that will contain NoteSequence protocol buffers.
 SEQUENCES_TFRECORD=/tmp/notesequences.tfrecord
 
 convert_dir_to_note_sequences \
-  --midi_dir=$MIDI_DIRECTORY \
+  --input_dir=$MIDI_DIRECTORY \
```
Comment (adarob) [inline magenta/scripts/README.md]: $INPUT_DIRECTORY

File: magenta/pipelines/README.md (hunk 1 above)
Comment (iansimon) [inline magenta/pipelines/README.md]: A DAGPipeline is also a Pipeline, I think it's clearer as is.

File: magenta/scripts/README.md (hunk 2 above)
Comment (iansimon) [inline magenta/scripts/README.md]: Fixed.

File: magenta/pipelines/README.md (hunk 1 above)
Comment (adarob) [inline magenta/pipelines/README.md]: How about changing the second `Pipelines` to "`DAGPipeline` objects"?

Comment (iansimon) [inline magenta/pipelines/README.md]: But only the outermost Pipeline is necessarily a DAGPipeline.  The component Pipelines can just be regular Pipelines.

Comment (iansimon) [inline magenta/pipelines/README.md]: Maybe I should change the first one to just clarify the whole thing, that it defines a DAGPipeline which is a subclass of Pipeline.

**Pull request merged**